src/bionics.cpp

```diff
@@ -1850,14 +1938,28 @@ void load_bionic( JsonObject &jsobj )
     bool faulty = jsobj.get_bool( "faulty", false );
     bool power_source = jsobj.get_bool( "power_source", false );
 
+    JsonArray jsarr;
+    std::map<body_part, size_t> occupied_bodyparts;
+    jsarr = jsobj.get_array( "occupied_bodyparts" );
+    if( !jsarr.empty() ) {
+        while( jsarr.has_more() ) {
+            JsonArray ja = jsarr.next_array();
+            occupied_bodyparts.emplace( get_body_part_token( ja.get_string( 0 ) ),
+                                        ja.get_int( 1 ) );
+        }
+    } else {
+        occupied_bodyparts.emplace( bp_torso, 0 );
```

Comment:
Okay, I think you're right. BTW, current code and your first solution causes this inappropriate note for CBMs listed above:
![bad_bp](https://cloud.githubusercontent.com/assets/5421954/14642391/5a802e4e-065b-11e6-956f-ad52c7bebf41.png)

For avoiding this, I'll implement your latter solution.

Thanks.